### PR TITLE
feat: add settings file to load environment variables

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR loads the environment variables from a `.env`. This file must be in `lambda/alexa/` folder

### CODE REVIEW

File: sample-skill/lambda/alexa/settings.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Settings for the Alexa skill."""
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_DOMAIN = os.getenv("API_DOMAIN")
```
Comment (mariajgrimaldi) [inline sample-skill/lambda/alexa/settings.py]: is this for eox-core? what do you think about EOX_CORE_API_DOMAIN? So it's more explicit.

File: sample-skill/lambda/alexa/settings.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+CLIENT_ID = os.getenv("CLIENT_ID")
+CLIENT_SECRET = os.getenv("CLIENT_SECRET")
+GRANT_TYPE = os.getenv("GRANT_TYPE")
```
Comment (mariajgrimaldi) [inline sample-skill/lambda/alexa/settings.py]: ^ Same comments here

**Review by mariajgrimaldi — APPROVED** [review]
Just minor suggestions!

File: sample-skill/lambda/alexa/settings.py (hunk 1 above)
Comment (BryanttV) [reply sample-skill/lambda/alexa/settings.py]: This is for edX and `eox-core` API, maybe a better name could be `LMS_DOMAIN`? What do you think?

**Review by Ian2012 — APPROVED** [review]

File: sample-skill/lambda/alexa/settings.py (hunk 1 above)
Comment (mariajgrimaldi) [reply sample-skill/lambda/alexa/settings.py]: LGTM!

**Pull request merged**